util/options_helper.h

```diff
@@ -688,6 +688,18 @@ static std::unordered_map<std::string, CompactionStyle>
         {"kCompactionStyleFIFO", kCompactionStyleFIFO},
         {"kCompactionStyleNone", kCompactionStyleNone}};
 
+static std::unordered_map<char, std::string> compaction_style_logging_map = {
+    {kCompactionStyleLevel, "level"},
```

Comment:
let's use the exact name of the enum as the string when logging
`kCompactionStyleLevel` instead of `level`
and so one